autopush/tests/test_websocket.py

```diff
@@ -54,6 +54,7 @@ def setUp(self):
         self.proto.sendClose = self.close_mock = Mock()
         self.proto.transport = self.transport_mock = Mock()
         self.proto.closeHandshakeTimeout = 0
+        self.proto.autoPingInterval = 300
```

Comment:
Well, I'm a fan of collecting metrics that answer questions rather than just bulk collection, but part of this exercise is that we really don't have much information backing up some of the decisions we're making. In this case, knowing how many aggressive pings we're shutting down shows how many clients we're effectively orphaning. These will be devices that will not have ping work correctly, and that's a concern. Hopefully it's a small percentage.

I'll agree that auditing other metrics is definitely worth it.